tests/std/tests/P0896R4_views_istream_death/test.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _CONTAINER_DEBUG_LEVEL 1
+
+#include <cassert>
+#include <cstddef>
+#include <ranges>
+#include <sstream>
+
+#include <test_death.hpp>
+using namespace std;
+
+using iview = ranges::basic_istream_view<int, char>;
+
+void test_predecrement_default_initialized() {
+    decltype(std::declval<iview>().begin()) it;

```

Comment:
It might be clearer to spell this type as `ranges::iterator_t<iview>`.

Reply:
But riddles?

changed